Assets/HoloToolkit-Examples/SpatialUnderstanding/SpatialUnderstanding-FeatureOverview/Scripts/SpatialUnderstandingBasicCursor.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using HoloToolkit.Unity.InputModule;
+using UnityEngine;
+
+/// <summary>
+/// 1. Decides when to show the cursor.
+/// 2. Positions the cursor at the gazed location.
+/// 3. Rotates the cursor to match hologram normals.
+/// </summary>
+public class SpatialUnderstandingBasicCursor : MonoBehaviour
```

Comment:
Looking at it, none of the scripts in the SpatialUnderstanding-FeatureOverview project are in a namespace. Given that, I'll leave SpatialUnderstandingBasicCursor in no namespace.

I agree that they should be changed so that each example is in its own namespace, but don't think it makes sense to do this work as part of the current pull request.